source/content/php-runtime-generation-2.md

```diff
@@ -128,6 +128,17 @@ If you encounter any issues while testing your site on the new PHP runtime gener
 
 Potentially. Depending on your integrations with our PHP extensions and operating system libraries, you may need to update your website to be compatible with new PHP runtime.
 
+
+### Q: How do I opt out of the upcoming platform rollout?
+
+You may opt-out of the upcoming platform rollout by setting the following in your `pantheon.yml`:
+
+```yaml:title=pantheon.yml
+php_runtime_generation: 1
+```
+
+Note: All sites will be auto-upgraded in Q1 2026 if they haven't already, including sites that specify the opt-out above.
```

Comment:
is this too opaque for people who don't speak in quarters? Also, is it too ambiguous (financial quarters -- which is what's generally referred to by Q1/2, etc -- might be different for different organizations/parts of the world)?